src/services/components/resources/highlight.json

```diff
@@ -1,3 +1,22 @@
 {
-  "name": "Highlight"
-}
+  "name": "Highlight",
+  "data": {
+    "text": "text",
+    "url": ""
```

Comment:
Please add sample text as in the default design so client can have reference to what kind of text he is writting